File: javaLabs/Shop/src/main/java/PaperGoods.java

public class PaperGoods extends Goods {
    private String titulPage;
    private PagesNumber pagesNumber;
    private PapersType paperType;

    public PaperGoods(String name, double price, int amount, ColorType color, CustumerType grade,
                      String titulPage, PagesNumber pagesNumber, PapersType paperType) {
        super(name, price, amount, color, grade);
        setTitulPage(titulPage);
        setPaperType(paperType);
        setNumber(pagesNumber);
    }

    public String getTitulPage() {
        return titulPage;
    }

    public void setTitulPage(String titulPage) {
        this.titulPage = titulPage;
    }

    public PapersType getPaperType() {
        return paperType;
    }

    public void setPaperType(PapersType paperType) {
        this.paperType = paperType;
    }

    public PagesNumber getPagesNumber() {
        return pagesNumber;
    }

    public void setNumber(PagesNumber pagesNumber) {
        this.pagesNumber = pagesNumber;
    }

    @Override
    public String toString() {
        return super.toString() + " titulPage = " + titulPage + ", pagesNumber = " + pagesNumber + ", paperType = " + paperType + "\n";
    }
<<<<<<< HEAD
    public String getHeaders() {
        return "name, price, amount, grade, color, titulPage pagesNumber, paperType ";
    }

    public String toCSV() {
        return getName() + ", " + getPrice() + ", " + getAmount() + ", " + getGrade() + ", " + getColorType() + ","+
                getTitulPage() + "," + getPagesNumber() + ", " + getPaperType() + ". ";
    }
=======
>>>>>>> f9adea0ec465bf3467f4510f80c04d0eafa792dd


}

File: python/GShop/Goods/PaperGoods.py
from Goods.Goods import Goods
from enums.PagesNumber import PagesNumber
from enums.PapersType import PapersType


class PaperGoods(Goods):
    pages_number = PagesNumber
    papers_type = PapersType

    def __init__(self, name, price, amount, custumer_type, color, pages_number, papers_type, titul_page):
        self.titul_page = titul_page
        self.pages_number = pages_number
        self.papers_type = papers_type
        Goods.__init__(self, name, price, amount, custumer_type, color)

    def __str__(self):
        return "Name : " + str(self.name) + " price: " + str(self.price) + "amount: " + str(self.amount)\
               + "color: " + str(self.color) + "custumer type: " + str(self.custumer_type) \
               + "titul page: " + str(self.titul_page) + "pages number: " + str(self.pages_number) \
               + "papers type: " + str(self.papers_type)